react-js/react.md

```diff
@@ -47,53 +50,65 @@ You will spend the day working through these materials.
 
 ### Topics to learn & Facilitate
 
-- What is React?
-  - React is a library for creating User Interfaces.
-  - This means it is used for creating what some refer to as the "view layer" of an application.
-- JSX adds to our JavaScript to allow for HTML-like markup in JavaScript.
-- Show Hello World example, plus expanded example.
-- Example for appending to a list in jQuery vs. the same in React.
+- React lets you generate HTML using JavaScript
+- The HTML can change over time, unlike regular HTML
+- No jQuery needed
+
+### Examples
+#### CodePen - Hello World component
+In this CodePen, a basic Hello World component is rendered.<br> 
```

Comment:
**Please remove all `<br>` from this doc**. It never belongs in markdown. Please find & replace before submitting your next revision.

To better learn GitHub markdown, see https://guides.github.com/features/mastering-markdown/

Reply:
Okay 👍